pkgs/development/python-modules/porto/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pypandoc
+, markdown
+, bleach
+, nbformat
+, jupyter-client
+, pygobject3
+, ipykernel
+, gtk3
+, webkitgtk
+, gtksourceview
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+# Added as a python package instead of an application
+# because in this way it can be easily added to a python environment
+# like this: python3.withPackages (p: with p; [ numpy porto ])
+# and I think in general this is easier to compose
```

Comment:
```suggestion
```

Reply:
You can add `porto = with python3.pkgs; toPythonApplication porto;` to `top-level/all-packages.nix`.

Reply:
@chuangzhu ^